include/envoy/router/rds.h

```diff
@@ -48,6 +48,17 @@ class RouteConfigProvider {
    * Callback used to notify RouteConfigProvider about configuration changes.
    */
   virtual void onConfigUpdate() PURE;
+
+  /**
+   * Callback used to request an update to the route configuration.
+   * @param for_domain supplies the domain name that virtual hosts contained in the VHDS response
```

Comment:
This is a bit confusing, as it is an update request, but the thing we're requesting is referencing the response..